README.md

```diff
@@ -21,7 +21,7 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+The file is a migrate file- it appears when you command rails migration and adds a new table to the database. The numbers are timestamps for the computer to know when the tables were created and prevent doubles from occurring and deleting information from the database.
 ```
```

Comment:
The command I believe you're thinking of is `rails g migration` or `rails generate migration`. Migrations can, but don't always, create a database table. They can also do things like add columns to existing tables.

You're getting there re: timestamps, but just to clarify, migrations use these timestamps to determine in what order they should be run relative to other migrations.